js/battle.js

```diff
@@ -3219,7 +3219,16 @@ var Battle = (function () {
 			if (kwargs.prepare || kwargs.anim === 'prepare') {
 				this.prepareMove(pokemon, move, target);
 			} else if (!kwargs.notarget) {
-				(kwargs.anim ? Tools.getMove(kwargs.anim) : move).anim(this, [pokemon.sprite, target.sprite]);
+				move = kwargs.anim ? Tools.getMove(kwargs.anim) : move;
```

Comment:
`getMove` just returns placeholder information in replays. It tells you the ID and gives you the string you passed in back out as a name. Basically, you can't read information from `moves.js` when you're in a replay, such as the move target.